motorlib/enums/units/densityUnit.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+# Python 3.11 supports `StrEnum` that would make this a bit more concise to write
+# https://docs.python.org/3/library/enum.html#enum.StrEnum
+class DensityUnit(str, Enum):
```

Comment:
I think I would prefer a single unit file, as it shouldn't be too huge.

Reply:
Done!
I left the names as "section comments" just to separate the Units. But if prefered it should be fairly easy to organize the units alphabetically (albeit it might look a bit weird, it might be easier to find)